docs/source/other_models/pregnancy/index.rst

```diff
@@ -1,9 +1,274 @@
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  ---------------------
+
+  Section Level 2 (#.#)
+  +++++++++++++++++++++
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
 =========================
 Pregnancy
 =========================
 
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - ASFR
+    - Age-specific fertility rate
+    - 
+  * - SBR
+    - Stillbirth to live birth ratio
+    - 
+  * - LBWSG
+    - Low birthweight and short gestation
+    - 
+
+Overview
+-------------
+
+.. warning::
+
+   The current version of this document details a strategy to model pregnancy in a simulation with a population of women of reproductive age only and does not yet detail a strategy to pair maternal child dyads.
+
+   Such a strategy will need to distinguish bewteen stillbirths and live births and assign the child's LBWSG exposure category in a way that is consistent with the mother's duration of pregnancy.
+
+GBD Modeling Strategy
+----------------------
+
+Pregnancy and births are not explit outcomes in the GBD study. However, there are location- and year-specfic GBD covariates related to fertility and births, including:
+
+.. list-table:: Covariates
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Covariate name
+    - Covariate ID
+    - Note
+  * - Live births (thousands)
+    - 60
+    - 
+  * - Live births by sex
+    - 1106
+    - 
+  * - Live births by sex and maternal age
+    - 2298
+    - 
+  * - Stillbirth to live birth ratio
+    - 2267
+    - Not specific to maternal age. Upper and lower bound estimates are equal to mean estimate (no uncertainty interval)
+  * - Age-specific fertility rate
+    - 13
+    - 
+  * - Total fertility rate
+    - 2363
+    - 
+
+GBD has estimated the prevalence of pregnancy (as in itermediate variable for the estimation of various outcomes) as:
+
+.. math::
+
+   (ASFR + (SBR * ASFR)) * 46/52
+
+Where :math:`ASFR` is the age-specific fertility rate, :math:`SBR` is the stillbirth to live birth ratio, and :math:`46/52` is the proportion of the year spent pregnant (40 weeks) and postpartum (6 weeks) [GBD-2019-Risk-Factors-Appendix-pregnancy]_.
+
 .. todo::
 
-   document this page
\ No newline at end of file
+   Determine the treshold of gestational age at which a loss of pregnancy is classified as a stillbirth rather than miscarriage for the GBD covariate. Standard thresholds are 20 or 24 weeks.
```

Comment:
```suggestion
   Determine the threshold of gestational age at which a loss of pregnancy is classified as a stillbirth rather than miscarriage for the GBD covariate. Standard thresholds are 20 or 24 weeks.
```